### ISSUE TITLE
Font spacing in between characters looks off

### ISSUE THREAD

**Issue by paaspaas00** [immich-app/immich#9529]
### The bug

The spacing in between characters is off, it is too much and a bit inconsistent. See here for instance 
![Screenshot_20240516-075622](https://github.com/immich-app/immich/assets/151774837/7a85f4e5-2e42-4218-997b-90fbf82fd202)
And compare for instance the bottom bar text to this:
![Screenshot_20240516-075819](https://github.com/immich-app/immich/assets/151774837/de50a1f6-1e03-48f7-b63c-91fd43931e9a)
As you can see, the text in immich app looks too widely spaced, in an inconsistent and weird way, I've no other way to describe it

### The OS that Immich Server is running on

Nn

### Version of Immich Server

Nn

### Version of Immich Mobile App

1.105.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Nn
```

### Your .env content

```Shell
Nn
```

### Reproduction steps

```bash
1.open app
2.look at fonts everywhere 
3.looks off off with respect for instance to another well known proprietary app people commonly use
```

**Comment by paaspaas00** [comment]
Looks almost like monospace, but with some difference between characters. Maybe it's an issue of the font used or maybe it's just some option to be set app-wise like in the android SDK apis

**Comment by paaspaas00** [comment]
May be related to https://github.com/flutter/flutter/issues/22572

**Comment by Fmstrat** [comment]
My two cents, I know it's branding, but the Immich font is way to "comic sans'esq". For such a great solution it's one of the few flaws in the app that makes it look less professional.

**Comment by alextran1502** [comment]
@Fmstrat any font that you can recommend? I once looked at a different font namely "Be Vietnam Pro" but hesitate to make the change

**Comment by Fmstrat** [comment]
@alextran1502 Well, my opinion is sheerly preference, and doesnt take whatever brand system you may have in mind into account, but since you ask:

My personal belief is a media app should fade into the background as much as possible, serving as an extension of the OS and enabling the media being viewed to create any sense of emotion and response. Because of this, I've personally always been a fan of just a plain `sans-serif-medium` interpreted by the OS. This was the route Fossify Gallery (previously SimpleMobile) and others have used, which was a nice touch to me.

**Comment by jrasm91** [comment]
If I'm not mistaken this was fixed in #5129
